doc/examples/R_double_ml_pipeline.ipynb

```diff
@@ -0,0 +1,790 @@
+    "    learner = lrn(\"classif.rpart\"))\n",
+    "\n",
+    "class(graph)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2ce3898",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "graph$plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11860a7e",
+   "metadata": {},
+   "source": [
+    "Create a new learner."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5e449f23",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "glrn = as_learner(graph)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "909193b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "glrn\n",
+    "class(glrn)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96a281d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "set.seed(123)\n",
+    "obj_dml_plr_bonus_pipe2 = DoubleMLPLR$new(dml_data_bonus,\n",
+    "                                          ml_g=ml_g_lasso,\n",
+    "                                          ml_m=glrn)\n",
+    "obj_dml_plr_bonus_pipe2$fit()\n",
+    "print(obj_dml_plr_bonus_pipe2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d552c2f",
+   "metadata": {},
+   "source": [
+    "Let's see how to set hyperparameters with a pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "27513f2e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "glrn$param_set$values$variance.filter.frac = 0.25"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3f35dca9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "set.seed(123)\n",
+    "obj_dml_plr_bonus_pipe3 = DoubleMLPLR$new(dml_data_bonus,\n",
+    "                                          ml_g=ml_g_lasso,\n",
+    "                                          ml_m=glrn)\n",
+    "obj_dml_plr_bonus_pipe3$fit()\n",
+    "print(obj_dml_plr_bonus_pipe3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f90db3e",
+   "metadata": {},
+   "source": [
+    "### Parameter tuning\n",
+    "\n",
+    "Next, we will shortly illustrate how to perform parameter tuning in the simulated data example. Here, we use a pipeline to tune the penalty of the lasso. To do so, we generate a [GraphLearner](https://mlr3pipelines.mlr-org.com/reference/mlr_learners_graph.html) and then call [DoubleML](https://docs.doubleml.org)'s [tune()](https://docs.doubleml.org/r/stable/reference/DoubleML.html#method-tune) method."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e1cf206d",
+   "metadata": {},
+   "source": [
+    "Let's define a [GraphLearner](https://mlr3pipelines.mlr-org.com/reference/mlr_learners_graph.html) based on the lasso."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "935daf2f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "graph_lasso = mutate %>>%\n",
```

Comment:
Not sure whether the variable name `graph_lasso` may result in misconceptions because there is also the Graphical Lasso. Maybe we better use something else, like for example `stacked_lasso`?

Reply:
I renamed it and called it `lasso_pipe` (which is in line with the previous pipeline based learners)